src/utils/getMessage.js

```diff
@@ -0,0 +1,23 @@
+import languages from '../i18n';
+import getAttr from './getAttr';
+import config from '../config';
+
+/* eslint no-param-reassign: ["error", { "props": false }]*/
+
+/**
+ * Uses getAttr to obtain translations for UI message strings outside react-intl.
+ * @param  {string} string - The identifier for the string to be localized
+ * @param  {string} lang - Language code for requested language
+ */
+
+const getMessage = (string, lang) => {
```

Comment:
Perhaps `lang` should just default to `config.language` (when it's moved there from `window`)?

```javascript
const getMessage = (string, lang=config.language) => { ... }
```
(unlike in Python, default arguments in ES6 are evaluated on every invocation; [see this babeljs transpilation for proof](https://babeljs.io/repl/#?babili=false&evaluate=false&lineWrap=false&presets=es2015&targets=&browsers=&builtIns=false&debug=false&experimental=false&loose=true&spec=false&code_lz=MYewdgzgLgBA5gUygWQRCBDRMC8MAU0ATgJZhwA0MANhuTqGAGYlwB0t5ArlggJS4AfDADeMAL4BuIA))